[2023-05-04 05:08:06,844] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.dim_product_line manual__2023-05-03T18:42:51.223967+00:00 [queued]>
[2023-05-04 05:08:07,098] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.dim_product_line manual__2023-05-03T18:42:51.223967+00:00 [queued]>
[2023-05-04 05:08:07,106] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-04 05:08:07,107] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-04 05:08:07,110] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-04 05:08:08,134] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): dim_product_line> on 2023-05-03 18:42:51.223967+00:00
[2023-05-04 05:08:08,204] {standard_task_runner.py:52} INFO - Started process 456 to run task
[2023-05-04 05:08:08,378] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'dim_product_line', 'manual__2023-05-03T18:42:51.223967+00:00', '--job-id', '910', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmpusagzmq2', '--error-file', '/tmp/tmp2s4pt_gx']
[2023-05-04 05:08:08,501] {standard_task_runner.py:80} INFO - Job 910: Subtask dim_product_line
[2023-05-04 05:08:11,623] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.dim_product_line manual__2023-05-03T18:42:51.223967+00:00 [running]> on host 295d97741f96
[2023-05-04 05:08:16,745] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=dim_product_line
AIRFLOW_CTX_EXECUTION_DATE=2023-05-03T18:42:51.223967+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-03T18:42:51.223967+00:00
[2023-05-04 05:08:18,752] {base.py:68} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-05-04 05:08:21,970] {python.py:173} INFO - Done. Returned value was: None
[2023-05-04 05:08:22,737] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=dim_product_line, execution_date=20230503T184251, start_date=20230504T050806, end_date=20230504T050822
[2023-05-04 05:08:23,436] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-04 05:08:25,499] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
